### DISCUSSION TITLE
SIM Swap Fraud Prevention

### DISCUSSION THREAD

**Discussion by heedfull** [authelia/authelia#7444] [General]
Hi,

I work for a mobile security company and we are in the early stages of developing a product to prevent SIM Swap Fraud for OpenID flows. We'd be very interested to have a conversation with you about this. If you are using SMS 2FA as part of your OpenID flow we may well be able to make it safer.

If you're happy to have a quick chat please let me know.

Many thanks,

Edwin

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

Hi Edwin,

Thank you for reaching out and expressing your interest in enhancing the security of our OpenID flows. Currently, Authelia does not utilize SMS for 2FA in its OpenID flow. Instead, we support several other methods for second-factor authentication, including Time-based One-Time Passwords (TOTP), WebAuthn Security Keys, and Duo Mobile Push notifications.

However, we are always open to exploring new ways to improve security. If you believe your product could offer additional benefits or if you have other suggestions, we would be happy to discuss this further. Please let us know your availability for a quick chat.

Looking forward to your response.

Best regards,

[Your Name]

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment]
We do not use SMS verification at all as identified by the bot.
